### PR TITLE
called_once_with should be assert_called_once_with

### PR DESCRIPTION
**Pull request by radez**
This is just a mistake?

### CODE REVIEW

**Review by Shrews — APPROVED** [review]

**Comment by Shrews** [comment]
Thanks for spotting this.

**Comment by radez** [comment]
@Shrews I'm not sure I'm familiar with the tests enough to understand the failure at first glance, would you mind taking a peek at the failure and see if you recognize what's failed?

File: test/unit/utils/test_dump_artifacts.py [hunk 1]
```diff
@@ -144,7 +144,7 @@ def test_dump_artifacts_inventory_object(mocker):
     kwargs = {'private_data_dir': '/tmp', 'inventory': inv}
     dump_artifacts(kwargs)
 
-    assert mock_dump_artifact.called_once_with(inv_string, '/tmp/inventory', 'hosts.json')
+    assert mock_dump_artifact.assert_called_once_with(inv_string, '/tmp/inventory', 'hosts.json')
```
Comment (Shrews) [inline test/unit/utils/test_dump_artifacts.py]:
```suggestion
    mock_dump_artifact.assert_called_once_with(inv_string, '/tmp/inventory', 'hosts.json')
```

**Review by Shrews — CHANGES_REQUESTED** [review]

**Comment by Shrews** [comment]
> @Shrews I'm not sure I'm familiar with the tests enough to understand the failure at first glance, would you mind taking a peek at the failure and see if you recognize what's failed?

Just need to remove the `assert` up front.

File: test/unit/utils/test_dump_artifacts.py (hunk 1 above)
Comment (radez) [reply test/unit/utils/test_dump_artifacts.py]: ah, yes that makes sense. I'll update. Thanks!

**Comment by Shrews** [comment]
FYI, codecov data is failing to upload, causing these test failures. I'll re-run them in a bit.

**Comment by radez** [comment]
Sounds good, thanks

**Review by Shrews — APPROVED** [review]

**Pull request merged**